### PR TITLE
Add header, footer and empty container to flatlist

### PR DESCRIPTION
**Pull request by Villar74**

### CODE REVIEW

File: src/index.tsx [hunk 1]
```diff
@@ -879,6 +887,9 @@ class DraggableFlatList<T> extends React.Component<Props<T>, State> {
         >
           <AnimatedFlatList
             {...this.props}
+            ListHeaderComponent={listHeader}
```
Comment (computerjazz) [inline src/index.tsx]: why is this necessary? props get spread onto the list, so you should just be able to pass `ListHeaderComponent` as a prop to `DraggableFlatlist`

Comment (Villar74) [reply src/index.tsx]: oh, i didn't notice that, lol, my bad
